### ISSUE TITLE
[Lee] Scene 3 Audio - Word change and other edits

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-people-use-web-videos#155]
[strong]
I think the word "required" is more common than "mandatory." If I am wrong, please feel free to not make that part of the changes below.

Current:

> Well, often only until I get to the sign-up or check-out forms for some of the online shops! I can’t tell you how often they say “fields in red are mandatory” or such. I prefer websites that indicate these fields in text as well as in color. For example, they put that little asterisk beside the mandatory fields or the word “optional” beside non-mandatory fields.

Suggestion:

> Another problem I find is when I get to a sign-up or check-out form for some of the online shops! I can’t tell you how often they say “fields in red are required". I prefer websites that indicate these fields in text as well as in color. For example, some put that little asterisk beside the required fields or the word “optional” beside non-required fields.

**Comment by MicheleAWilliams-A11y** [comment]
I'll add here that I found it hard to follow the "Well, often only until...". 

I think this was supposed to transition from online shopping being easier than in-person and this scene is presenting a monkey wrench in that. Thus, I would suggest the following (building on what Brent said):

> One thing that _is_ difficult about online shopping though is when I get to the checkout and it says "fields in red are required". Ugh! Most times I can't tell this apart from the regular text so I have to hope I catch everything. It's much better when they either have the word "required" or that little asterisk beside the fields, or even better put the word "optional" besides the ones I can skip.

**Comment by nitedog** [comment]
Changed to:

> Unfortunately, other shops aren't as good. Often the sign-up or checkout form says "fields in red are required". Ugh! Most times I can't tell which fields are required when they use color only. Good shops either have the word "required" or that little asterisk beside the fields, or even better put the word "optional" besides the ones I can skip.
